recipes/genesis2/bld.bat

```diff
@@ -0,0 +1,18 @@
+copy mpi.f.single mpi.f
+copy mpif.h.single mpif.h	
+
+x86_64-w64-mingw32-gfortran -g -w -O -Wall -o genesis2 main.f check.f diagno.f esource.f field.f incoherent.f math.f partsim.f pushp.f loadbeam.f loadrad.f magfield.f tdepend.f track.f string.f rpos.f scan.f source.f stepz.f timerec.f initrun.f  input.f output.f mpi.f > output.txt 2>&1
```

Comment:
I think we need some `if errorlevel 1 exit 1` lines after this and other steps. Maybe there is some error we are missing that this will surface.

Reply:
I added after every step to make sure. Once the recipe is going good for windows I will clean up this script and remove the debug print statements as well. Thanks for the hint on the errorlevel.

Reply:
Yeah this is analogous to `set -e` in `bash`. Unfortunately `batch` script lacks a way to set this for the entirety of the script. So we just have to add that line after every command.

Reply:
Got it! But looks like somehow it is stuck again while creating the test environment. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=225510&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77

Reply:
What if we try just calling the `build.sh` script from `bld.bat` [like so]( https://github.com/conda-forge/minikube-feedstock/blob/32d3a940e8f46b66e7184d4cad7be5090f6bbeb8/recipe/bld.bat#L1-L8 )? We would also want to add `posix` as a `build` dependency on Windows [as done here]( https://github.com/conda-forge/minikube-feedstock/blob/32d3a940e8f46b66e7184d4cad7be5090f6bbeb8/recipe/meta.yaml#L19 ). This way we are doing the exact same thing on Windows as Unix. Hopefully that eliminates whatever might be causing us issues here.

Reply:
It doesn't hurt to try... Should I still keep the same compilers for windows based on `m2w64`?

Reply:
Trying it via https://github.com/conda-forge/staged-recipes/pull/12889/commits/702728085044855c89f5fd03817f8873034753a0

Reply:
Yeah let's give that a try.

Does the `Makefile` produce some kind of log that we can `cat` if it fails?

Reply:
The makefile won't produce a log... if error happens we will catch it from the dump to `stdout`.
The build failed. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=225527&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=734
I will add some logging tomorrow and resume the tests.
Thanks for the time and help so far! :+1:

Reply:
Ah ok. Ofc! Hopefully the logging yields some insights :)

Reply:
@jakirkham I finally got the windows build to work with `posix` but yet again I am having issues with the conda environment creation for the tests.
Is there a way to enable debug flags for the test step? It seems to me that it is conda failing to create the environment more than an issue with the package itself.

Since both are resulting in the same effect for the windows build I will revert to the non-posix build.

Reply:
Let's use the bat signal 🦸 🦇 

@conda-forge/core, we seem to be running into an issue where Conda-build hangs creating the test environment. Has anyone seen this before? Is there something we are doing wrong here? Any suggestions? 🙂